src/app/providers/react-query.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { Suspense } from "react"
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query"
+
+export const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      refetchOnWindowFocus: false,
+    },
+  },
+})
+
+interface ReactQueryProps {
+  children: React.ReactNode
+}
+
+const ReactQuery = (props: ReactQueryProps) => {
+  return (
+    <Suspense>
```

Comment:
Not sure that Suspense should be placed here. It should be imported and used in a file that bundles up all providers.

Reply:
Done